src/client/driver/command-executors/execute-action-command.js

```diff
@@ -66,6 +67,8 @@ function runAutomation (element, command) {
         automation = new RClickAutomation(element, command.options);
     else if (command.type === COMMAND_TYPE.doubleClick)
         automation = new DblClickAutomation(element, command.options);
+    else if (command.type === COMMAND_TYPE.hover)
+        automation = new HoverAutomation(element, command.options);
 
```

Comment:
We decide to use just `switch` for this case because we'll have different arguments for different command in future. I'll fix it